euclidean_gcd/C++/euclidean_gcd.cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+using namespace std;
+
+int gcd(int a, int b){
+	if (a == 0)
+		return b;
+	return gcd(b % a, a); // returns gcd(remainder, a) if gcd(a,b) = r then gcd(a,b) = gcd(b,r)
```

Comment:
Lines should be <= 80 characters long  [whitespace/line_length] [2]

*Origin: CPPLintBear, Section: `all.cpplint`.*